instance/models/instance.py

```diff
@@ -119,11 +123,13 @@ class Instance(ValidateModelMixin, models.Model):
     # in a query, e.g. to do .select_related('ref_set')
     ref_set = GenericRelation(InstanceReference, content_type_field='instance_type', object_id_field='instance_id')
     openstack_region = models.CharField(
+        # TODO: Move this to OpenedXInstance
         max_length=16,
         blank=False,
         default=default_setting('OPENSTACK_REGION'),
     )
     tags = models.ManyToManyField(
+        # TODO: Move this to InstanceReference (common fields should be in non-abstract tables where possible)
```

Comment:
Argh, you're right, disregard what I said about the `DRY` issue - in my haste to reply I confused which model this was being added to.

> Querying tags across different instance types feels rather contrived to me

I actually considered making `is_archived` simply an `archived` tag, which would be an example of this kind of thing that applies to all instance types.

> That said, I'm not really opposed to moving the tags to the instance reference. I just don't see strong arguments either way.

OK, I'm fine to leave this for now and see how we tend to use these things in practice. Later on, we'll know more about what our priorities are. I'll remove my opinionated comment.